src/app/components/Charts/BarChart.tsx

```diff
@@ -0,0 +1,70 @@
+import { useGetBarChartDataQuery } from "@/app/redux/features/api";
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  BarElement,
+  Title,
+  Tooltip,
+  Filler,
+  Legend,
+} from "chart.js";
+import { Bar } from "react-chartjs-2";
+import { formatDateLabel } from "./utils";
+
+ChartJS.register(
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  BarElement,
+  Title,
+  Tooltip,
+  Filler,
+  Legend
+);
+
+const options = {
+  scales: {
+    y: {
+      display: false,
+    },
+    x: {
+      display: true,
+      ticks: {
+        maxTicksLimit: 8,
+        color: "#9b9ab6",
+      },
+      grid: {
+        display: false,
+      },
+    },
+  },
+  elements: {
+    point: {
+      radius: 0,
+    },
+    line: {
+      tension: 1,
+    },
+  },
+};
+
+const BarChart = () => {
+  const { data } = useGetBarChartDataQuery("");
+  const barChartLabels = data?.volume.map((el) => formatDateLabel(el[0]));
+  const barChartData = {
+    labels: barChartLabels,
+    datasets: [
+      {
+        label: "",
```

Comment:
check if you need this

Reply:
![Screenshot 2024-03-03 at 10 43 47 AM](https://github.com/napqueenkaila/crypto-app/assets/125321591/fb8c0e7d-7d0e-41e6-a076-59019404a7d9)

without the empty label property it will render "undefined" on the chart